doc/source/Tutorials/ManagingComplexity/index.md

```diff
@@ -48,7 +48,7 @@ PathFilter Wildcards
 
 The `'...'` wildcard in PathFilter expressions means "match any number of names". So, `'/world/.../house'` matches `'/world/village/house'` as well as `'/world/country/state/metropolis/neighbourhood/street/house'`. This can be very useful, but it comes at a price. Certain operations in Gaffer require the scene hierarchy to be searched for all matches below a particular location. As you might guess, the presence of `'...'` forces this search to go ever deeper, just in case a child (or a child of a child of a child) might match. For large scenes, this can become very expensive.
 
-The most painful match expression possible is `'/.../something'`, because it literally says "search the entire scene". Where possible then, searches should at least be limited by using a prefix such as `'/AssetA/.../something'`, restricting the search to just everything below `'AssetA'`. In general, the more restricted the search, the better the performance.
+The most painful expression possible is `'/.../something'`, because it literally says "search the entire scene". Although this can be necessary at times, experience has shown that such expressions are often used where a more precise form could do the same job with better performance. For instance, if it is known that all the matches are within a single asset, an expression such as `/AssetA/.../something` will limit the search to that asset only. Alternatively, if it known that all the matches are at a specific depth, a pattern such as `/*/something` or `/*/*/something` will yield the same results without needing to visit deeper locations. Small changes such as this not only have a significant impact on scene performance.
```

Comment:
Trailing off? "Small changes such as this not only have a significant impact on scene performance."